src/@utils/aquarius/index.ts

```diff
@@ -41,6 +41,18 @@ export function escapeEsReservedCharacters(value: string): string {
  */
 type TFilterValue = string | number | boolean | number[] | string[]
 type TFilterKey = 'terms' | 'term' | 'match' | 'match_phrase'
+type Query = {
+  must: {
+    exists: {
+      field: string
+    }
+  }
+  must_not?: {
+    term: {
+      [key: string]: string
+    }
+  }
```

Comment:
suggestion(blocking): This seems to have a lot of duplication with `interface BoolFilter`. Could be suitable to create another interface which is reused in `interface BoolFilter` and here.